README.md

```diff
@@ -24,27 +24,44 @@ To add ECS exporter to your existing ECS task:
 1. Click on "Create new revision".
 1. Scroll down to "Container definitions" and click on "Add container".
 1. Set "ecs-exporter" as container name.
-1. Copy the container image URL from above.
+1. Copy the container image URL from above. (Use the tag for the [latest
+   release](https://github.com/prometheus-community/ecs_exporter/releases).)
 1. Add tcp/9779 as a port mapping.
 1. Click on "Add" to return back to task definition page.
 1. Click on "Create" to create a new revision.
 
 By default, it publishes Prometheus metrics on ":9779/metrics". The exporter in this repo can be a useful complementary sidecar for the scenario described in [this blog post](https://aws.amazon.com/blogs/opensource/metrics-collection-from-amazon-ecs-using-amazon-managed-service-for-prometheus/). Adding this sidecar to the ECS task definition would export task-level metrics in addition to the custom metrics described in the blog.
 
-The sidecar process is also supported on [AWS App Runner](https://aws.amazon.com/apprunner/)
-and can be used to publish infra metrics in Prometheus format
-from App Runner services.
+## Compatibility guarantees
+All metrics exported by ecs_exporter are sourced from the [ECS task metadata
+API](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-metadata-endpoint-v4.html)
+accessible from within every ECS task. AWS can make, and on occasion [has
+made](https://github.com/prometheus-community/ecs_exporter/issues/74#issuecomment-2395293862),
+unannounced (or at least unversioned) breaking changes to the data served from
+this API, especially the container-level stats served from the `/task/stats`
+endpoint. Metrics emitted by ecs_exporter may spontaneously break as a result,
+in which case we may need to make breaking changes to ecs_exporter to keep up.
+
+TODO what is our versioning policy in light of this situation? Do breaking
+changes to metric names/semantics/labels even constitute major version bumps in
+other exporters, let alone those in this situation?
```

Comment:
@SuperQ can you offer any guidance on what to say here?  Are there any other prominent exporters in this situation that we can mimic?